blocks/carousel/block.php

```diff
@@ -23,6 +24,8 @@
     'autoHeight' => $auto_height,
     'autoplay' => $auto_play ? ['delay' => 5000] : false,
     'loop' => $loop,
+    'simulateTouch' => true,
```

Comment:
@x3blondie I've found it best to set shortSwipes to true when using simulate touch: https://swiperjs.com/swiper-api#param-shortSwipes